doc/cephadm/monitoring.rst

```diff
@@ -14,8 +15,27 @@ performance.  Ceph users have three options:
 #. Skip the monitoring stack completely.  Some Ceph dashboard graphs will
    not be available.
 
-Deploying monitoring with cephadm
----------------------------------
+The monitoring stack consists of `Prometheus <https://prometheus.io/>`_,
+Prometheus exporters (:ref:`mgr-prometheus`, `Node exporter
+<https://prometheus.io/docs/guides/node-exporter/>`_), `Prometheus Alert
+Manager <https://prometheus.io/docs/alerting/alertmanager/>`_ and `Grafana
+<https://grafana.com/>`_.
+
+.. note::
+
+  Please note that Prometheus' security model presumes that untrusted users
+  have access to the Prometheus HTTP endpoint and logs. They have access to
+  all time series information contained in the database, plus a variety of
```

Comment:
What's "all time series information"?

Reply:
"all time series information" is all the (meta)data Prometheus collects about machines and the Ceph cluster through Prometheus exporters, namely Prometheus manager module and Node exporter (in our current case).

I took the sentence from the Prometheus documentation and I'm not sure if it's necessary to go into that detail here, though I may be wrong.

Reply:
:thinking: hmm, so it is the name of the data. Although without the context it sort of makes it seem.. well... odd. What do you think of this:

"Untrusted users have access to all the (meta)data Prometheus collects contained in the database..." (you could then explain that the data is called "all time series information" but the direct answer works fine, provides more context for those who haven't had a lot to do with Prometheus. I do leave that up to your call, the audience here should be aware, but you never know.